apps/src/templates/instructions/TopInstructions.jsx

```diff
@@ -144,7 +144,7 @@ class TopInstructions extends Component {
         (teacherViewingStudentWork && this.props.noInstructionsWhenCollapsed
           ? teacherViewingStudentTab
           : TabType.INSTRUCTIONS),
-      feedbacks: [],
+      feedback: null,
```

Comment:
Is this now just storing the latest feedback?  If so, would calling this `latestFeedback` be a little clearer?